interface/patient_file/summary/demographics.php

```diff
@@ -392,6 +392,26 @@ function toggleIndicator(target,div) {
       'centerOnScroll' : false
     }).trigger('click');
     <?php } ?>
+<?php /*To display the birthday alert
+    1. The patient is not dead
+    2. The birthday is today
+    3. The notification has not been turned off for this year*/
+
+    if ($GLOBALS['patient_birthday_alert']) {
+        include_once ('../birthday_alert/birthday.inc.php');
```

Comment:
use `require_once`
For organization sake, recommend just placing this include at the top with the other includes.